styles.css

```diff
@@ -552,3 +631,189 @@ button:active {
 ::placeholder {
   color: #979493;
 }
+
+@media only screen and (min-width: 768px) {
+  .menu-icon {
+    display: none;
+  }
+
+  .nav-bar-pages {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    color: #fff;
+    text-decoration: none;
+    justify-self: right;
+    margin: 15px;
+  }
+
+  .headline {
+    flex-direction: row-reverse;
+    align-items: center;
+    padding: 200px 10px 130px 10px;
+  }
+
+  .social-media-icons {
+    flex-direction: column;
+    margin-right: 0;
+  }
+
+  footer .social-media-icons {
+    flex-direction: row;
+    margin-right: 0;
+  }
+
+  .headline-info {
+    align-items: flex-start;
+  }
+
+  .description {
+    width: 45%;
+  }
+
+  .background-img1 {
+    background-image: url('media/illustration-heading.png');
+    background-repeat: no-repeat;
+    position: absolute;
+    width: 430.89px;
+    height: 365.03px;
+    left: 65%;
+    top: 100px;
+    transform: scale(1.8);
+  }
+
+  .work1 {
+    display: grid;
+    grid-template-columns: 1.5fr 1fr;
+    grid-template-rows: 1fr;
+    width: 90%;
+    height: auto;
+    justify-items: left;
+    align-items: flex-start;
+    max-width: 1100px;
+  }
```

Comment:
Your project cards look great. However, you seem to be missing some detail from the figma design. The card with the light background signifies how card should look when it is hovered. Kindly add this to your design.

Figma design:
![image](https://user-images.githubusercontent.com/20554583/206505623-53c06f1f-af8e-41dd-bdc5-f697471c91db.png)